evidence/skunert/0001-skunert-retain-II-2024.md

```diff
@@ -0,0 +1,42 @@
+# Evidence-0001: Retention at Rank II
+
+|                  |                  |
+| ---------------- | ---------------- |
+| **Report Date**  | 2024/05/03       |
+| **Submitted by** | Sebastian Kunert |
+
+## Member details
+
+- Matrix username: `@sebastian:parity.io`
+- Polkadot address: `1682A5hxfiS1Kn1jrUnMYv14T9EuEnsgnBbujGfYbeEbSK3w`
+- Current rank: 2
+- Date of initial induction: 2024/01/08
+- Date of last report: N/A
+- Area(s) of Expertise/Interest: `Cumulus Node`, `Substrate Node`
+
+## Reporting period
+
+- Start date: 2024/01/08
+- End date: 2024/05/13
+
+## Evidence
+
```

Comment:
https://github.com/polkadot-fellows/RFCs/pull/43 your rfc is missing :see_no_evil:

Reply:
I think the RFC was before the current time period :thinking: . At least opening time, I was not sure what counts.

Reply:
I would include it. I mean you worked on it in the reported time.